components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -146,6 +162,31 @@ service ReportingService {
 	};
 }
 
+message ControlItemRequest {
+	string text = 1;
+	int32 size = 2;
+	repeated ListFilter filters = 3;
+}
+
+message StringTag {
+	string key = 1;
+	repeated string values = 2;
+}
+
+message ControlItem {
+	string id =1;
+	string title=2;
+	ProfileMin profile = 3;
+	float impact = 4;
+	google.protobuf.Timestamp end_time = 5;
+	ControlSummary control_summary = 6;
+	repeated StringTag string_tags = 7;
+}
+
+message ControlItems {
+	repeated ControlItem control_item = 1;
```

Comment:
could we maybe call this controls?  as in `repeated ControlItem controls = 1;` ?
or discuss with ux?  i feel like it sounds weird to have a singular "item" for something that is going to be an array.
```
{
  "control_item": [
    {
      "id": "tmp-3.0",
      "title": "Create /tmp directory",
      "profile": {
        "name": "",
        "title": "InSpec Tag Test Profile",
        "id": "1e93c77aca38eeffa495637011e74bf615628cf6c46fad05e067bbc16b895c6a",
        "version": "0.2.0",
        "status": ""
      },
...
```

Reply:
I don't really view these as simply controls.. they are more than that. Maybe ControlStats?
But also, they are also bit specific.  For the given filter and the latest run in our timeseries data:
1.  They provide stats.
2.  They provde the tags specific.

With them being so specific, I guess I just figured that calling them "control" was not enough.

Reply:
let's ask ux how they feel?  i don't necessarily care about the naming, it's the lack of pluralization that gets me